### ISSUE TITLE
Top Racer Collection

### ISSUE THREAD

**Issue by RealRelativeEase** [farmerbb/RED-Project#152]
QuByte recently released the Top Racer collection, consisting of several Top Gear/Top Racer games. The SNES ROMs can be found under: `TopRacerCollection_Data/StreamingAssets/`

TR1.smc = Top Gear/Top Racer
TR2.smc = Top Gear 2/Top Racer 2
TR3.sfc = Top Gear 3000/Top Racer 3000
TRC.smc = Top Gear Crossroads/Top Racer Crossroads

Crossroads appears to be an enhanced version of the first game, exclusive to this collection. FYI, Steam also has a demo of the collection.

**Comment by TheNametag** [comment]
Crossroads appears to be one where it is patched real time in the game.  The ROM on its own doesn't appear functional.

**Comment by vaguerant** [comment]
Indeed. See ares-emulator/ares#1420.

It's not as fun as legally grabbing the ROM out of your purchased collection, but it's worth mentioning that a version of *Crossroads* without the anti-piracy functionality leaked to the Internet a couple of months ago under the name *Top Gear Crossroads Horizon Chase*, which can be used to play the hack outside of Top Racer Collection.

**Comment by RealRelativeEase** [comment]
What emulator have you been using to play the patched ROM? I'm still getting corrupted graphics or a black screen in SNES9X when trying to start a race. The patched ROM has a SHA1 checksum of `a3753f7c986ffb4a37fc7fe3bd4b39cf6c98b1bf`.

**Comment by RealRelativeEase** [comment]
> > What emulator have you been using to play the patched ROM? I'm still getting corrupted graphics or a black screen in SNES9X when trying to start a race. The patched ROM has a SHA1 checksum of `a3753f7c986ffb4a37fc7fe3bd4b39cf6c98b1bf`.
> 
> My SHA1 checksum is `7e742f3de39ef83ee33ccbc763ed6753d6e3ec23`. Your checksum is for the unpatched ROM.

Yeah, I messed up during the patching process. Game runs fine now, thanks for the patch!
